worldguard-core/src/main/java/com/sk89q/worldguard/session/handler/FlagValueChangeHandler.java

```diff
@@ -45,6 +45,12 @@ public final void initialize(LocalPlayer player, Location current, ApplicableReg
         onInitialValue(player, set, lastValue);
     }
 
+    @Override
+    public final void uninitialize(LocalPlayer player, Location current, ApplicableRegionSet set) {
+        onAbsentValue(player, current, current, set, lastValue, MoveType.OTHER_NON_CANCELLABLE);
```

Comment:
Not the most beautiful thing. We could also add a method like `onClearValue(LocalPlayer, ApplicableRegionSet)` for "completeness" where the default implementation would call back to `onAbsentValue` to not break existing plugins.

Reply:
I definitely prefer having a dedicated clear, gives more flexibility to things that need it for additional cleanup etc.